arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -173,6 +173,125 @@
         }
       }
     },
+    "/{resourceId}/providers/Microsoft.Authorization/locks/{lockName}": {
```

Comment:
can we please replace {resourceId} with {scope}. In ARM specs, resourceId signifies the fully qualified Id of a resource